.github/CODEOWNERS

```diff
@@ -462,6 +462,7 @@ cypress.config.js @grafana/grafana-frontend-platform
 /public/app/routes/ @grafana/grafana-frontend-platform
 /public/app/store/ @grafana/grafana-frontend-platform
 /public/app/types/ @grafana/grafana-frontend-platform
+/public/app/types/alerting.ts @grafana/alerting-frontend
```

Comment:
Sorry for the ping folks, probably makes sense for @grafana/alerting-frontend to own this